packages/runtime/render/src/server/ssr/ssr.tsx

```diff
@@ -0,0 +1,82 @@
+import type {
+  ClientManifest,
+  SSRManifest,
+  SSRModuleMap,
+} from '@modern-js/types/server';
+import type { ReactNode } from 'react';
+import type { ReactDOMServerReadableStream } from 'react-dom/server';
+import { renderToReadableStream } from 'react-dom/server.edge';
+import { ServerElementsProvider } from '../../client';
+
+type Options = {
+  request: Request;
+  clientManifest: ClientManifest;
+  ssrManifest: SSRManifest;
+} & Parameters<typeof renderToReadableStream>[1];
+
+function wrapStream(
```

Comment:
what's the usage of this function.

Reply:
In order to assign the properties of the original stream to the new stream, the return value of renderToReadableStream is `ReactDOMServerReadableStream`, not the original `ReadableStream`.